packages/env/lib/env.js

```diff
@@ -97,6 +103,31 @@ module.exports = {
 			// Some commit refs need to be set as detached.
 			await repo.setHeadDetached( ref );
 		}
+
+		// Duplicate repo for the tests container.
+		let stashed = true; // Stash to avoid copying config changes.
```

Comment:
I wonder if, rather than this stash + filesystem copy, we could just create a clone from the `repoPath`.

In some brief testing in the Terminal, when I'm in `gutenberg-wordpress`, then run:

```
git clone . ../gutenberg-tests-wordpress/
```

... it creates a checkout of WordPress, at the same commit as in `gutenberg-wordpress`, excluding modifications in the working directory. Isn't this the same as what you're trying to achieve with the logic here?

Reply:
I don't think it copies over things like a detached head ref from `await repo.setHeadDetached( ref );`.